docs/getting-started-guides/fedora/fedora_ansible_config.md

```diff
@@ -46,11 +44,15 @@ Get the IP addresses from the master and minions.  Add those to the inventory fi
 192.168.121.205
 
 [minions]
-192.168.121.84  kube_ip_addr=[ignored]
-192.168.121.116 kube_ip_addr=[ignored]
+192.168.121.84  kube_ip_addr=[10.254.0.1]
+192.168.121.116 kube_ip_addr=[10.254.0.2]
 ```
 
-**Tell ansible which user has ssh access (and sudo access to root)**
+**Setup ansible access to youre nodes**
```

Comment:
one more.... since I can't merge until tomorrow anyway, might as well fix this one too.